src/components/Onboarding/content.js

```diff
@@ -104,8 +104,8 @@ highlights.rinkeby = [
       small: null,
       large: (
         <span>
-          When you vote in favor of the majority ruling, you will be rewarded
-          with arbitration fees. Just note that these rewards will be awarded in
+          When you vote in favor of the plurality ruling, you will be rewarded
```

Comment:
I'm not opposed to it, but we've used "majority" in quite a few places to describe this mechanism.

And while I agree "plurality" is the technically more correct term, it's also less familiar to non-English speakers.

Reply:
> I'm not opposed to it, but we've used "majority" in quite a few places to describe this mechanism.

I already removed this term from all of the help.aragon.org documentation (e.g. [this section](https://help.aragon.org/article/41-aragon-court#plurality)) because it is not correct. It was a mistake to ever describe Aragon Court as a majority-rule system. This PR also fixes the terminology throughout the Dashboard.

> And while I agree "plurality" is the technically more correct term, it's also less familiar to non-English speakers.

The other equally-accurate term would be to say "relative majority", which I decided against because 1) most people would also have to look up what that means, and 2) it's two words and that takes up more space and feels awkward to write over and over again: 

> Aragon Court is a _relative majority-rule_ system.... You have ruled with the _relative majority_... Make sure to vote the way that you believe the _relative majority_ of jurors will vote...

vs

> Aragon Court is a plurality-rule_ system.... You have ruled with the _plurality_... Make sure to vote the way that you believe the _plurality_ of jurors will vote...